### ISSUE TITLE
card_size has no effect

### ISSUE THREAD

**Issue by JoeyG1973** [twrecked/lovelace-hass-aarlo#105]
I have tried to set card_size to all kinds of positive numbers and no matter what the size of card always remains the same.

**Comment by twrecked** [comment]
Agreed. I included it because the developer documents told me to but I can't get it to make much of a difference.

I need to change the base JavaScript class I use so I'll see if it makes a difference then.
